hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ReplicatedMapCodecTemplate.java

```diff
@@ -49,7 +49,7 @@
      * @param name Name of the ReplicatedMap
      * @return the number of key-value mappings in this map.
      */
-    @Request(id = 2, retryable = true, response = ResponseMessageConst.INTEGER)
+    @Request(id = 2, retryable = true, response = ResponseMessageConst.INTEGER, partitionIdentifier = "random")
     Object size(String name);
```

Comment:
I am not sure if "random" is the right word here. It implies that with each request you can just send a different random ID. What happens in reality is it picks one at random and then always uses that one..